napi/parser/package.json

```diff
@@ -43,8 +43,7 @@
     "wasm.mjs",
     "bindings.js",
     "webcontainer-fallback.js",
-    "deserialize-js.js",
-    "deserialize-ts.js"
+    "generated"
```

Comment:
This breaks https://github.com/oxc-project/oxc/blob/78cdb1c5252cef5e888600243e71566ebf28d415/.github/workflows/reusable_release_napi.yml#L174-L176

Reply:
I've made this change:

```diff
- "generated"
+ "generated/deserialize/js.js",
+ "generated/deserialize/ts.js"
```

I think that fixes it?

Reply:
@Boshen I think this is good to go now?